src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/MainPanel.spec.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {shallowWithTheme} from '@jahia/test-framework';
+import {dsGenericTheme} from '@jahia/design-system-kit';
+
+import {MainPanel} from './MainPanel';
+
+describe('PickerDialog - MainPanel', () => {
+    let props;
+    beforeEach(() => {
+        props = {
+            pickerConfig: {
+                picker: {
+                    PickerDialog: {
+                        dialogTitle: () => 'yoloooooo',
+                        DialogContent: 'DialogContent',
+                        searchPlaceholder: () => 'searchPlaceholder'
+                    }
+                }
+            },
+            nodeTreeConfigs: [{
+                type: 'image'
+            }],
+            editorContext: {
+                lang: 'fr'
+            },
+            selectedPath: '/selectedPath',
+            setSelectedPath: jest.fn(),
+            searchTerms: '',
+            handleSearchChange: jest.fn(),
+            onItemSelection: jest.fn(),
+            onCloseDialog: jest.fn(),
+            t: arg => arg
+        };
+    });
+
+    it('should display the title set in the picker configuration', () => {
+        const cmp = shallowWithTheme(
+            <MainPanel {...props}/>,
+            {},
+            dsGenericTheme
+        ).dive();
+
+        expect(cmp.debug()).toContain('');
```

Comment:
Question: I don't understand the case, where is the title that should be displayed?

Reply:
You are right this test is wrong 😨